apps/flighttracker/flighttracker.star

```diff
@@ -71,7 +71,7 @@ def main(config):
     airportCode = config.get("airportCode") or DEFAULTAIRPORT
 
     # Date utilities for the API calls. These dates are not utilized in the render.
-    now = time.now().in_location("America/New_York")
+    now = time.now().in_location("Europe/London")
```

Comment:
In case this was meant to be GMT, note that London is currently in BST. You can use "Etc/UTC" here for GMT.